### DISCUSSION TITLE
[01_Head_count] ログ出力について

### DISCUSSION THREAD

**Discussion by moriyaks** [Ignitarium-Renesas/RZV2L_AiLibrary#7] [Q&A] [answered]
[01_Head_count]について
画面に表示されている頭の数といった情報をCSV等へログとして出力することは可能でしょうか。
例）以下のような3つのデータのログ出力
ログＮｏ．　処理時間　頭のカウント数
1：100ms　5人
2：70ms　  2人

**Comment by mohitb-ign** [comment, marked as answer]
Thank you for this question.

Yes, it is possible to output the log in csv format.
Already we are displaying string output on the video feed. That string output can be written in a file.

Relevant source code is available at  01_Head_count/Head_count_cam/src/sample_app_yolov3_cam.cpp - line number 614, Function name 'int8_t print_result( )'
